src/components/Badges/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { bem } from '../../../utils';
+import { Context } from '../../../constants';
+import { BadgeVariant } from '../../../constants/component-specific';
+import styles from './StatusBadge.scss';
+
+export interface Props extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
```suggestion
export interface Props extends React.HTMLAttributes<HTMLSpanElement> {
```
Since `rest` is applied to a Span not a Div